include/exec/repeat_effect_until.hpp

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) 2023 NVIDIA Corporation
```

Comment:
@Runner-2019 you should add your own copyright here as well. On a separate line.

Reply:
Yes, thank you. I added a copyright notice to the latest submission